### DISCUSSION TITLE
Is it possibble not to move files at all?

### DISCUSSION THREAD

**Discussion by isle9** [beetbox/beets#5088] [Q&A] [answered]
Is it possible to keep my current path for my files, so that they are not grouped in albums and artists (they are still grouped under artists if you use singleton)?
I am using Navidrome and any path change deletes the song from Navidrome's database.

**Comment by hg2s** [comment]
Does the "[update](https://beets.readthedocs.io/en/v1.6.0/reference/cli.html?highlight=beet%20update#update)" command solve this problem for you? There is a dry-run option so you can test it.

**Reply by isle9** [reply]
"Update the library (and, by default, move files)". From the wording, seems like this would only work if I have my beets library already set up? Although it does mention moving files in the brackets. I'll try using the dry run option to check what happens.

**Comment by hg2s** [comment]
I know this has been addresses before but I don't recall what it was called. It might be that in order to suppress the 'move' you have to use a different configuration file that has something like 'move: no' in it. Of course you will need to have your singleton path specified as well. 

Otherwise, have you read the [Command Line Interface](https://docs.beets.io/en/latest/reference/cli.html#update) notes on 'reimporting' and 'update'? It isn't clear to me what the -a flag does in 'update'. Maybe you can test it and report back.

**Reply by isle9** [reply]
I re-read some of the documentation stuff, and it seems that I just actually need to use copy: no in the config and it should leave my directory completely untouched? I can then run the importer on my existing music directory, and that should be it?

**Comment by hg2s** [comment]
Did it work for you?

**Reply by isle9** [reply]
I haven't tried it yet, but iirc, it should work.

**Reply by JOJ0** [reply, marked as answer]
Set copy to no.
And move to no.

Then you can select on import and modify commands manually when you want to copy/move

beet import --copy

beet import --move

beet mod --copy

beet mod --move

For newly downloaded stuff I often use --move.

For existing stuff that shouldnt be altered I dont use either option, ie my default is to never touch path!!
